tests/integration/test_data_cli.py

```diff
@@ -599,7 +605,12 @@ def test_data_search_create_filter_success(invoke, item_types):
     respx.post(TEST_SEARCHES_URL).return_value = mock_resp
 
     runner = CliRunner()
-    result = invoke(["search-create", name, item_types, json.dumps(filter)],
+    result = invoke([
+        "search-create",
+        item_types,
+        f'--name={name}',
+        f'--filter={json.dumps(filter)}'
```

Comment:
A little nit, but for consistency with the function's input, you could change the order of inputs to `item_types, search_filter, name`, which you already do for the test `test_data_search_create_daily_email()`
```suggestion
        item_types,
        f'--filter={json.dumps(filter)}',
        f'--name={name}'
```